SPECS/ocaml-gnt.spec

```diff
@@ -1,8 +1,8 @@
 %define debug_package %{nil}
 
 Name:           ocaml-gnt
-Version:        1.0.0
-Release:        2%{?dist}
+Version:        2.2.2
+Release:        1%{?dist}
 Summary:        OCaml bindings for userspace Xen grant table controls
 License:        LGPL2.1 + OCaml linking exception
```

Comment:
I see the LICENSE file has dropped off. However I also see from the source tree that the license is no longer LGPL-mumble... It's now ISC: https://github.com/mirage/ocaml-gnt/blob/master/_oasis#L6